test/ERC7914.t.sol

```diff
@@ -0,0 +1,208 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.23;
+
+import {DelegationHandler} from "./utils/DelegationHandler.sol";
+import {IERC7914} from "../src/interfaces/IERC7914.sol";
+import {ERC7914} from "../src/ERC7914.sol";
+import {IERC7821} from "../src/interfaces/IERC7821.sol";
+
+contract ERC7914Test is DelegationHandler {
+    event TransferFromNative(address indexed from, address indexed to, uint256 value);
+    event ApproveNative(address indexed owner, address indexed spender, uint256 value);
+    event ApproveNativeTransient(address indexed owner, address indexed spender, uint256 value);
+    event TransferFromNativeTransient(address indexed from, address indexed to, uint256 value);
+
+    address alice = makeAddr("alice");
```

Comment:
nit use diff name i think the signerAccount address is 0xa11ce which might end up being confusing